src/test/collectionHandlingFunctions.php

```diff
@@ -0,0 +1,314 @@
+<?php
+require "../collectionHandlingFunctions.php";
+use PHPUnit\Framework\TestCase;
+
+class collectionHandlingFunctions extends TestCase 
+{
+
+    public function testFailureDescribeCollectionItem_EmptyItem() 
+    {
+        $yarnExample = [];
+        $this->expectException(Exception::class); 
+        describeCollectionItem($yarnExample);
+    }
+    
+    public function testFailureDescribeCollectionItem_undefinedName() {
+        $yarnExample = ["name" => NULL, 
```

Comment:
You should put the first array item on the next line after the [, indented by one level.

Here's an example of what I mean

```
$example = [
    "foo" => "bar",
    "bar" => "baz"
];
```

Reply:
This applies to the other arrays in this test file

Reply:
done!